Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM nginx:1.17
+
+COPY . /usr/share/nginx/html
+COPY config-sample.json /usr/share/nginx/html/config.json
+COPY docker/docker-entrypoint.d /docker-entrypoint.d
+COPY docker/entrypoint.sh /
+COPY docker/run.sh /run.sh
+COPY docker/nginx.conf.template /etc/nginx/conf.d/default.conf.template
```

Comment:
Please create a `root` folder to copy resources and then you only need to `COPY root /` in a single docker layer